modules/ingesting-content-into-a-llama-model.adoc

```diff
@@ -170,12 +188,12 @@ document = RAGDocument(
 +
 [source,python]
 ----
-client.tool_runtime.rag_tool.insert(
+client.vector_stores.files.create(
```

Comment:
I think this was also mentioned here: https://issues.redhat.com/browse/RHOAIENG-37887